transcom/mymove
# MTO shipment risk of excess banner persistence solution

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19178)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
create a move => submit as SC => approve as TOO => login as prime, locate the move, create a shipment => login as 

1) Create a move as a services counselor and submit.
2) Approve the move as a TOO
3) Login as a prime user and create a shipment under that move with weight values.
4) Login as TOO and approve the new shipment then navigate to the move task order tab.
If the weight is at risk of excess, there should be a warning banner at the top of the dashboard.
5) Click the  'x' on the banner. When the page is refreshed, or accessed from that point forward, the banner should not re-appear.
6) You can verify the qualification time in the db with
`select excess_weight_qualified_at  from moves where locator  = 'GQCKFH'`
### Frontend


### Backend



## Screenshots
